cmd/infra/powervs/destroy.go

```diff
@@ -373,10 +388,7 @@ func destroyPowerVsCloudInstance(ctx context.Context, options *DestroyInfraOptio
 		return err
 	}
 
-	if options.CloudInstanceID != "" {
-		// In case of user provided cloud instance delete only DHCP server
-		err = destroyPowerVsDhcpServer(ctx, infra, cloudInstanceID, session, options.InfraID)
-	} else {
+	if options.CloudInstanceID == "" {
```

Comment:
Lets have early return by
```suggestion
	if options.CloudInstanceID != "" {
	    log(options.InfraID).Info("skipping the cloud instance because user supplied")
	    return nil
	}
```

Reply:
Done